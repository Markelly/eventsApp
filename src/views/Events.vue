<template>
  <div class="events">
    <h1 data-events-title>EVENTS</h1>
    <EventsList :showAllEvents="true" data-events-list />
  </div>
</template>

<script>
import EventsList from "../components/EventsList";
export default {
  name: "Events",
  components: { EventsList }
};
</script>

<style scoped lang="scss">
@import "../styles/events-app";
</style>
