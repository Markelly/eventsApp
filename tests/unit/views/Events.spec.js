import Events from "../../../src/views/Events";
import { createLocalVue, shallowMount } from "@vue/test-utils";

describe("Cards", () => {
  const localVue = createLocalVue();
  let wrapper;

  test("should render the component", () => {
    wrapper = shallowMount(Events, { localVue });
    expect(wrapper).toMatchSnapshot();
  });
});
