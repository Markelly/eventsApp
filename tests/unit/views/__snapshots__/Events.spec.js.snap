// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards should render the component 1`] = `
<div class="events">
  <h1 data-events-title="">EVENTS</h1>
  <eventslist-stub data-events-list=""></eventslist-stub>
</div>
`;
